src/assets/scss/application.scss

```diff
@@ -290,4 +290,38 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  float: right;
+  border-bottom: 1px solid $govuk-border-colour;
+  .govuk-list {
```

Comment:
In our component model, we try to adhere to a BEM way of writing CSS instead of relying on nesting selectors. 

If you need to style the unordered list element, instead of relying on specificity to overwrite existing rules, we would instead assign another class to `ul` in addition to `govuk-list`. So the `ul` becomes `<ul class="govuk-list language-select__list">`. The links become `<a class="{{insert govuk link classes here}} language-select__link">`. And so on.

So then the SCSS can be changed to:

```
.language-select {
 ... styles specific to the NAV container...
}

.language-select__list {
... styles that are specific to the UL element only
}

.language-select__link {
... styles that are specific to the anchor tag element only
}
```

and so on.

Let me know if any of that doesn't make sense.

Reply:
All done and sorted